Makefile

```diff
@@ -1,9 +1,53 @@
 TOOL_VERSION = 2.1.12
 TEST_DATA = test/simpleTestForGenerator
-
+GIT_AUTHOR = License Publisher (maintained by Gary O'Neall) <gary@sourceauditor.com>
+LICENSE_DATA_REPO_NO_SCHEME = github.com/goneall/license-list-data.git
+LICENSE_DATA_REPO = https://$(LICENSE_DATA_REPO_NO_SCHEME)
+LICENSE_DATA_URL = https://$(GITHUB_TOKEN)@$(LICENSE_DATA_REPO_NO_SCHEME)
+LICENSE_OUTPUT_DIR = .tmp
+VERSION = $(shell git describe --always || echo 'UNKNOWN')
+RELEASE_DATE = $(shell date '+%Y-%m-%d')
+COMMIT_MSG = License list build $(VERSION) using license list publisher $(TOOL_VERSION)
+RELEASE_MSG = Adding release matching the license list XML tag $(VERSION)
+#TODO Change the license data repo to license-list-xml before merging pull request
```

Comment:
I'm not sure what you intend to do here, but if you're getting close to merging, you may want to address this todo.

Reply:
I'll update the repo url after the travis run and right before the merge

Reply:
> I'll update the repo url after the travis run and right before the merge

Looks like you updated the URL in 7d169ff8164, but you still have this comment?